dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -100,7 +101,18 @@
         .smalltext= body
       = render partial: 'shared/courseblock_action', locals: { id: id, showState: showState, url: url, current_user: current_user }
 
--elsif [Script::COURSE1_NAME, Script::COURSE2_NAME,Script::COURSE3_NAME, Script::COURSE4_NAME, Script::ALGEBRA_NAME].include? id
+-elsif [Script::EXPRESS_NAME, Script::PRE_READER_EXPRESS_NAME].include? id
+  - height = 120
+  .courseblock-span6.courseblock-wide-small{style: "min-height: #{height}px;"}
+    =link_to link_to script_url(id)  do
+      .courseblock-span2.imgspan
+        %img{src: CDO.code_org_url("/shared/images/courses/fill-150x110/logo_tall_#{id}.jpg")}
+      .courseblock-span4.heading
+        %h3= data_t_suffix('script.name', id, 'title')
+        .smalltext= data_t_suffix('script.name', id, 'description_short')
+      .bottombar.smalltext.audience= data_t_suffix('script.name', id, 'description_audience')
```

Comment:
I would love to do this.  There seems to be a bunch of special-casing in the HAML which will take a while to digest, and I know even what we're building in this PR is going to change in in the next couple weeks as the sets of blocks we show changes.  But if @tanyaparker can get time on the schedule, I'm all for rewriting these in React.

In the meantime, one goal of this change was to at least pull the management of these blocks into a single set of React components, so it's a baby step on the journey.  

It's a pretty last-minute change, and needs to go out tomorrow for the launch of courses A-F, so its scope is limited to the new courses and the refactor.